tests/pipeline/test_uncertainty_qbin_pipeline.py

```diff
@@ -0,0 +1,496 @@
+    "ComparisonBins/testing_compQ_PHD-NET_E-MHA_4CH_combinedFalse_recall_jaccard_all_lms.pdf",
+    "ComparisonBins/testing_compQ_PHD-NET_E-MHA_4CH_combinedFalse_precision_jaccard_all_lms.pdf",
+    "ComparisonBins/testing_compQ_PHD-NET_E-MHA_4CH_combinedFalsejaccard_lm_0.pdf",
+    "ComparisonBins/testing_compQ_PHD-NET_S-MHA_4CH_combinedFalse_undotted_error_all_lms.pdf",
+    "ComparisonBins/testing_compQ_PHD-NET_S-MHA_4CH_combinedFalse_undotted_error_lm_0.pdf",
+    "ComparisonBins/testing_compQ_PHD-NET_S-MHA_4CH_combinedFalse_undottedmean_error_folds_all_lms.pdf",
+    "ComparisonBins/testing_compQ_PHD-NET_S-MHA_4CH_combinedFalse_errorbound_all_lms.pdf",
+    "ComparisonBins/testing_compQ_PHD-NET_S-MHA_4CH_combinedFalse_errorbound_lm_0.pdf",
+    "ComparisonBins/testing_compQ_PHD-NET_S-MHA_4CH_combinedFalse_jaccard_all_lms.pdf",
+    "ComparisonBins/testing_compQ_PHD-NET_S-MHA_4CH_combinedFalse_recall_jaccard_all_lms.pdf",
+    "ComparisonBins/testing_compQ_PHD-NET_S-MHA_4CH_combinedFalse_precision_jaccard_all_lms.pdf",
+    "ComparisonBins/testing_compQ_PHD-NET_S-MHA_4CH_combinedFalsejaccard_lm_0.pdf",
+    "ComparisonBins/testing_compQ_U-NET_E-MHA_4CH_combinedFalse_undotted_error_all_lms.pdf",
+    "ComparisonBins/testing_compQ_U-NET_E-MHA_4CH_combinedFalse_undotted_error_lm_0.pdf",
+    "ComparisonBins/testing_compQ_U-NET_E-MHA_4CH_combinedFalse_undottedmean_error_folds_all_lms.pdf",
+    "ComparisonBins/testing_compQ_U-NET_E-MHA_4CH_combinedFalse_errorbound_all_lms.pdf",
+    "ComparisonBins/testing_compQ_U-NET_E-MHA_4CH_combinedFalse_errorbound_lm_0.pdf",
+    "ComparisonBins/testing_compQ_U-NET_E-MHA_4CH_combinedFalse_jaccard_all_lms.pdf",
+    "ComparisonBins/testing_compQ_U-NET_E-MHA_4CH_combinedFalse_recall_jaccard_all_lms.pdf",
+    "ComparisonBins/testing_compQ_U-NET_E-MHA_4CH_combinedFalse_precision_jaccard_all_lms.pdf",
+    "ComparisonBins/testing_compQ_U-NET_E-MHA_4CH_combinedFalsejaccard_lm_0.pdf",
+    "ComparisonBins/testing_compQ_U-NET_S-MHA_4CH_combinedFalse_undotted_error_all_lms.pdf",
+    "ComparisonBins/testing_compQ_U-NET_S-MHA_4CH_combinedFalse_undotted_error_lm_0.pdf",
+    "ComparisonBins/testing_compQ_U-NET_S-MHA_4CH_combinedFalse_undottedmean_error_folds_all_lms.pdf",
+    "ComparisonBins/testing_compQ_U-NET_S-MHA_4CH_combinedFalse_errorbound_all_lms.pdf",
+    "ComparisonBins/testing_compQ_U-NET_S-MHA_4CH_combinedFalse_errorbound_lm_0.pdf",
+    "ComparisonBins/testing_compQ_U-NET_S-MHA_4CH_combinedFalse_jaccard_all_lms.pdf",
+    "ComparisonBins/testing_compQ_U-NET_S-MHA_4CH_combinedFalse_recall_jaccard_all_lms.pdf",
+    "ComparisonBins/testing_compQ_U-NET_S-MHA_4CH_combinedFalse_precision_jaccard_all_lms.pdf",
+    "ComparisonBins/testing_compQ_U-NET_S-MHA_4CH_combinedFalsejaccard_lm_0.pdf",
+]
+
+
+def test_qbin_pipeline():
+    """Test the uncertainty quantile binning pipeline."""
+
+    # ---- setup configs ----
+    cfg = get_cfg_defaults()
```

Comment:
is it necessary to use .yaml for configuration in test?

Reply:
Fixed